mlflow/models/model.py

```diff
@@ -75,6 +76,7 @@
 SET_MODEL_ERROR = (
     "Model should either be an instance of PyFuncModel, Langchain type, or LlamaIndex index."
 )
+ENV_VAR_FILE_NAME = "environment_variables.txt"
```

Comment:
Is there a reason that we don't write environment variables in `MLmodel`?

Reply:
Initially I'm afraid there would be too many env vars and it could make MLmodel file messy

Reply:
Good point. One con of isolating this file is that another network request is required to get this file. The request might fail depending on the mlflow version used for logging the model. If `MLmodel` includes environment variables, we just fetch it and parse it.

Reply:
I feel like MLmodel file contains a bunch of information, and a separate file could be clearer, and we check if the file exists before reading the context, WDYT?

Reply:
I don't have a strong preference at this point. A separate file seems ok.

Reply:
I think a separate file is fine. Can we file a follow-up to update our doc for the artifact structure?
https://mlflow.org/docs/latest/models.html#id13
It misses many recent additions e.g. metadata folder, input example (+ its serving counterpart), and this new one.